pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>org.eclipse.smarthome</groupId>
     <artifactId>smarthome-packaging-sample</artifactId>
-    <version>0.9.1-SNAPSHOT</version>
+    <version>0.9.2-SNAPSHOT</version>
```

Comment:
Why not `0.10.0-SNAPSHOT`?

Reply:
As ESH is yet at 0.9 ?

Reply:
Ah, sure, as long as we are using the release it makes sense.

```
<esh.version>0.9.0</esh.version>
```